kits/pharmacy-refill-assistant/agent.md

```
@@ -0,0 +1,29 @@
+# Pharmacy Refill Assistant
+
+## Overview
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)